py/pysparkling/ml/params.py

```diff
@@ -48,21 +77,36 @@ def setWeightCol(self, value):
         assert_is_type(value, str)
         return self._set(weightCol=value)
 
+    def setSeed(self, value):
+        assert_is_type(value, int)
+        return self._set(seed=value)
+
+    def setNfolds(self, value):
+        assert_is_type(value, int)
+        return self._set(nfolds=value)
+
+    def setAllStringColumnsToCategorical(self, value):
+        assert_is_type(value, bool)
+        return self._set(allStringColumnsToCategorical=value)
+
+    def setColumnsToCategorical(self, value):
+        assert_is_type(value, [str])
```

Comment:
Shouldn't we also expose an overloaded method with a variable number of parameters?

Reply:
done